lib/gooddata/rest/connection.rb

```diff
@@ -9,7 +9,7 @@ module GoodData
   module Rest
     # Wrapper of low-level HTTP/REST client/library
     class Connection
-      DEFAULT_URL = 'https://secure.gooddata.com'
+      DEFAULT_URL = ENV['GD_GEM_SERVER'] || 'https://secure.gooddata.com'
```

Comment:
`GD_GEM_SERVER` is a bit weird name - I first though that this is some GoodData's rubygems server. Maybe just `GD_SERVER` or `GD_URL`?